aws-analytics-pinpoint/src/androidTest/java/com/amplifyframework/analytics/pinpoint/AnalyticsPinpointInstrumentedTest.java

```diff
@@ -59,6 +66,19 @@ public static void setUp() throws AmplifyException {
         configuration.populateFromConfigFile(context, R.raw.amplifyconfiguration);
         Amplify.addPlugin(new AmazonPinpointAnalyticsPlugin());
         Amplify.configure(configuration, context);
+        plugin = (AmazonPinpointAnalyticsPlugin) Amplify
```

Comment:
AmazonPinpointAnalyticsPlugin plugin = new AmazonPinpointAnalyticsPlugin();
Amplify.configure(plugin);

// Now you can reference it using the reference `plugin`